playbooks/roles/l2tp-ipsec/tasks/main.yml

```diff
@@ -120,12 +120,14 @@
 
 - name: Include the L2TP/IPsec sysctl value variables
   include_vars: sysctl-values.yml
+  when: ansible_virtualization_type != 'lxc'
 
 - name: Apply the sysctl values
   sysctl:
     name: "{{ item.key }}"
     value: "{{ item.value }}"
   with_items: "{{ l2tp_ipsec_sysctl_values }}"
+  when: ansible_virtualization_type != 'lxc'
```

Comment:
Instead of putting two identical `when` conditions on the "Include" and "Apply" sysctl tasks could you make a block with both of these tasks inside of it and put the when condition on the block?